### ISSUE TITLE
Feather select, ARIA-controls can point to non-existent element

### ISSUE THREAD

**Issue by jmillar-nanthealth** [feather-design-system/feather-design-system#107]
We are getting failures in AXE tests because the `ARIA-controls` attributes contain invalid values.
It appears the ID included does not exist on the page as there was a change to Feather Menu to use `v-if`.

We should change the ARIA-controls to only be active when the dropdown in open

**Comment by jmillar-nanthealth** [comment]
This is resolved through PR #108
